lib/runner/cli/nightwatch.conf.ejs

```diff
@@ -184,8 +184,8 @@ module.exports = {
       // https://www.browserstack.com/automate/capabilities?tag=selenium-4
       desiredCapabilities: {
         'bstack:options' : {
-          userName: '${BROWSERSTACK_USER}',
-          accessKey: '${BROWSERSTACK_KEY}',
+          userName: '<%- browserstackUserName %>',
+          accessKey: '<%- browserstackAccessKey %>',
```

Comment:
No needed

Reply:
this and accompanying the change in `cli.js` was done in case no conf file has been provided by the user, in that case nightwatch will generate the `conf` file on first test run and would add `'${BROWSERSTACK_KEY}'` into the conf file which in turn would lead to the accessKey being set as `process.env.BROWSERSTACK_KEY,` and give an error even if `process.env.BROWSERSTACK_ACCESS_KEY` has been set.

Reply:
We don't need need `process.env.BROWSERSTACK_ACCESS_KEY`

Reply:
please remove all of those changes adding additional process envs

Reply:
> We don't need process.env.BROWSERSTACK_ACCESS_KEY
> please remove all of those changes adding additional process envs
The entire idea behind this PR is to add support for accepting these env variables as these are the standard env variable names used in browserstack's documentation..

Reply:
We can change that in local config simply by adding `${BROWSERSTACK_USERNAME}`

Reply:
The use case this is solving is: User has set `BROWSERSTACK_USERNAME` in env variables but has not passed anything in conf file.
Currently nightwatch works if `BROWSERSTACK_USER` is set but nothing is sent in conf file.
Would want `BROWSERSTACK_USERNAME` to behave the same way.

Reply:
and similarly `BROWSERSTACK_ACCESS_KEY` as `BROWSERSTACK_KEY` behaves currently.